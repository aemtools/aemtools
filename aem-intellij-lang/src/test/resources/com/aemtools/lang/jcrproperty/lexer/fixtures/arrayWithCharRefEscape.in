[function check() {&#xd;&#xa; return &#34;&#34;;&#xd;&#xa;},function check2() {&#xd;&#xa; return &#34;&#34;;&#xd;&#xa;}]
