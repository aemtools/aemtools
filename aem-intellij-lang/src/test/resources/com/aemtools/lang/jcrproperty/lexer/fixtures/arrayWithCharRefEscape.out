[ ('[')
ARRAY_VALUE_TOKEN ('function check() {')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xd;')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xa;')
ARRAY_VALUE_TOKEN (' return ')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#34;')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#34;')
ARRAY_VALUE_TOKEN (';')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xd;')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xa;')
ARRAY_VALUE_TOKEN ('}')
, (',')
ARRAY_VALUE_TOKEN ('function check2() {')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xd;')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xa;')
ARRAY_VALUE_TOKEN (' return ')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#34;')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#34;')
ARRAY_VALUE_TOKEN (';')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xd;')
VALID_XML_CHAR_REF_ESCAPE_TOKEN ('&#xa;')
ARRAY_VALUE_TOKEN ('}')
] (']')
